.github/test-matrix.json

```diff
@@ -0,0 +1,110 @@
+{
+  "include": [
```

Comment:
The unfortunate thing about this file is that it's JSON

- JSON is insanely frustrating to edit by hand. And this seems to be a file edited by hand. make it a yaml
- JSON precludes comments, and file like this wants comments. 
  - first of all, it should start with a comment explaining what it is
  - we will also want to have a way to add inline comments

Reply:
Done!